.appveyor.yml

```diff
@@ -21,6 +21,8 @@ before_build:
   - 7z x go%GO_VERSION%.windows-amd64.zip -oC:\ >nul
   - go version
   - choco install codecov
+  # Print host version. Remove this when containerd has a way to get host version
+  - $psversiontable
```

Comment:
Looks like you have to tell it to use powershell? try adding the `ps: ` prefix (ref: https://www.appveyor.com/docs/build-configuration/#powershell)

Reply:
Yep 😅 just noticed that